jupyterlab_gcedetails/src/data.ts

```diff
@@ -140,4 +140,32 @@ export const STYLES = stylesheet({
     display: 'table-row',
     boxShadow: 'inset 0 -1px 0 0 var(--jp-border-color0)',
   },
+  chartTitleSmall: {
+    fontSize: '20px',
+    marginLeft: '20px',
+  },
 });
+
+export const AREA_CHART_BLUE = {
```

Comment:
Should the styles related consts in this file be moved to the style folder?

Reply:
Hmm in general in this repo styles are defined in the typescript files and the styles folder is just used to store and define images but I don't have a strong opinion on this. By the way if the specific constants you defined here only apply to a single component its probably better to just define them in the corresponding components file instead of defining and exporting from here

Reply:
Yea true, I was thinking of having the color ones here in case they're needed again, but it might be better to export just the hex for the color profiles, unless its already done elsewhere?

Reply:
If you mean general shared colors profiles across this repo, some are defined here: https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/master/shared/src/styles.ts#L32 ?